src/seedu/addressbook/data/tag/Tagging.java

```diff
@@ -0,0 +1,21 @@
+package seedu.addressbook.data.tag;
+
+import seedu.addressbook.data.person.*;
+
+public class Tagging {
+	
+	private final Person person;
+	private final Tag tag;
+	private String sign;
+	
+	public Tagging(Person person, Tag tag, String sign) {
+		this.person = person;
+		this.tag = tag;
+		this.sign = sign;
+	}
```

Comment:
Please fix the indentation and change the tab to spaces in Eclipse, see [here](http://stackoverflow.com/questions/407929/how-do-i-change-eclipse-to-use-spaces-instead-of-tabs)